### ISSUE TITLE
Plugin not working in Notepad++ v7.6.6

### ISSUE THREAD

**Issue by cridus** [sieukrem/jn-npp-plugin#74]
As the title says.
Since other plugins weren't detected by Notepad++ with this version, I tried doing the same thing that fixed the problem for those: putting the dll in a folder with the exact same name (without the extension). It still doesn't work (it appears in the "plugins" menu but the scripts aren't working and not even the "about" works), and when closing Notepad++ it gives an "access violation" error.
I can't live without the SmartHighlighter, pls halp! D:

(By the way is there any log file I can check for errors?)

**Comment by trdm** [comment]
working in Notepad++ v6.8.2

**Comment by sieukrem** [comment]
I did not get any plugin installed via plugin manager v7.6.6 x64.  Also if I uninstall the preinstalled Npp Converter plugin from Don Ho I am not able to install it again.

So I assume there is an issue inside of Npp release.

Why do you need the latest version of Npp?

**Comment by deasserted** [comment]
I upgraded to 7.6.6 (from 7.5.9) because the auto-updater prompted it, and it looked like there were some worthwhile security fixes.  But like the OP, I haven't been able to get jN working.  I used NPP's Settings -> Import -> Import Plugin(s) to add jN.dll.  But none of my .js files seem to be working.  

I value jN enough I'll downgrade NPP to get jN back.  But it sounds like NPP's developer thinks he is done with fixing plugin bugs.  He may need some direct interaction or all future versions might not support jN.

Thanks for the incredibly useful tool!

**Comment by cridus** [comment]
> I did not get any plugin installed via plugin manager v7.6.6 x64. Also if I uninstall the preinstalled Npp Converter plugin from Don Ho I am not able to install it again.
> 
> So I assume there is an issue inside of Npp release.
> 
> Why do you need the latest version of Npp?

Well I don't necessarily _need_ it, but since the other plugins work when moved each inside its own folder and this one doesn't, I guess there is indeed a problem with this one. But yeah, I guess I'll roll-back and wait to see if it turns out that the bug is actually in Npp.

**Comment by sieukrem** [comment]
Do you use zip deployment or do you install npp?
Do you use  x64 or x86?

**Comment by deasserted** [comment]
For me: I install x86 (not zip)

**Comment by cridus** [comment]
I used the installer and I guess I have the x86 version since it installs itself in the "Program Files (x86)" folder.

**Comment by dvkolesnikov** [comment]
For me the plugin stopped working when I updated to NPP 7.6.6 (64 bit).
I tried to remove the jN.dll and jN folder from the "plugins" folder then to download the last available zip and to extract it to the "plugins" folder again but that didn't help (of course I restarted NPP).
Then I found "jN" in Plugins admin (Plugins -> Plugins Admin) and installed it from there. That helped for me and now the plugin works well. Hope it will help to someone.

**Comment by inu20** [comment]
I'm just about to comment. =D

> For me the plugin stopped working when I updated to NPP 7.6.6 (64 bit).
> I tried to remove the jN.dll and jN folder from the "plugins" folder then to download the last available zip and to extract it to the "plugins" folder again but that didn't help (of course I restarted NPP).
> Then I found "jN" in Plugins admin (Plugins -> Plugins Admin) and installed it from there. That helped for me and now the plugin works well. Hope it will help to someone.

**Comment by cridus** [comment]
> For me the plugin stopped working when I updated to NPP 7.6.6 (64 bit).
> I tried to remove the jN.dll and jN folder from the "plugins" folder then to download the last available zip and to extract it to the "plugins" folder again but that didn't help (of course I restarted NPP).
> Then I found "jN" in Plugins admin (Plugins -> Plugins Admin) and installed it from there. That helped for me and now the plugin works well. Hope it will help to someone.

Thanks, now it works! How could I not think about just trying to reinstall it? 🤦‍♂️

**Comment by deasserted** [comment]
Thanks for the tips -- it is nice to have that problem behind me now.  I think the NPP plugin manager was my problem; it seemed to not have updated correctly (or maybe I overwrote it with an older version).

BORING DETAILS: For me, reinstall did not work.  Uninstall using the plugin manager did not work (NPP still thought it was installed).  Manually moving the files out of the plugin directory did not word (NPP still thought it was installed).  I began to suspect the plugin manager was the problem -- maybe I had the old one from before NPP made all these changes that messed up the plugins.

I finally succeeded by downloading NPP 7.6.6 again, in zip format, and used it to overwrite whatever NPP  files I thought I had not personalized (made a backup copy, just in case).  After that, when I opened NPP's plugin manager, it looked different (whiter background).  NPP finally said that jN was not installed, so I install jN as a plugin, and jN worked again.

Thanks to all who shared their paths to success.
